dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/MetadataValueRestRepository.java

```diff
@@ -0,0 +1,226 @@
+                                                   @Parameter(value = "qualifier", required = false)
+                                                                  String qualifierName,
+                                                   @Parameter(value = "searchValue", required = false) String
+                                                                  searchValue,
+                                                   Pageable pageable) {
+        if (StringUtils.isBlank(searchValue)) {
+            throw new DSpaceBadRequestException("searchValue cannot be null!");
+        }
+
+        Context context = obtainContext();
+
+        String separator = ".";
+        String metadataField = StringUtils.isNotBlank(schemaName) ? schemaName + separator : "";
+        metadataField += StringUtils.isNotBlank(elementName) ? elementName : "";
+        metadataField += StringUtils.isNotBlank(qualifierName) && !StringUtils.equals(UNDEFINED, qualifierName) ?
+                separator + qualifierName : "";
+
+        List<String> metadata = List.of(metadataField.split("\\."));
+        // metadataField validation
+        if (StringUtils.isNotBlank(metadataField)) {
+            if (metadata.size() > 3) {
+                throw new IllegalArgumentException("Query param should not contain more than 2 dot (.) separators, " +
+                        "forming schema.element.qualifier");
+            }
+        }
+
+        if (searchValue.contains(":")) {
+            searchValue = searchValue.replace(":", "");
+        }
+
+        // Find matches in Solr Search core
+        DiscoverQuery discoverQuery =
+                this.createDiscoverQuery(metadataField, searchValue, pageable);
+
+        List<MetadataValueWrapper> metadataValueWrappers = new ArrayList<>();
+        try {
+            DiscoverResult searchResult = searchService.search(context, null, discoverQuery);
+            for (IndexableObject object : searchResult.getIndexableObjects()) {
+                if (object instanceof IndexableItem) {
+                    // get metadata values of the item
+                    List<MetadataValue> metadataValues = itemService.getMetadataByMetadataString(
+                            ((IndexableItem) object).getIndexedObject(), metadataField);
+
+                    // convert metadata values to the wrapper
+                    List<MetadataValueWrapper> metadataValueWrapperList =
+                            this.convertMetadataValuesToWrappers(metadataValues);
+                    metadataValueWrappers.addAll(metadataValueWrapperList);
+                }
+            }
+        } catch (SearchServiceException e) {
+            log.error("Error while searching with Discovery", e);
+            throw new IllegalArgumentException("Error while searching with Discovery: " + e.getMessage());
+        }
+
+        // filter eu sponsor -> do not return eu sponsor suggestions because for eu sponsors is used
```

Comment:
"... suggestions for items where eu sponsor is used."

Reply:
fixed